### ISSUE TITLE
Javadoc escapes no longer work properly

### ISSUE THREAD

**Issue by dlew** [square/retrofit#1719]
For example, see [`QueryMap`](https://square.github.io/retrofit/2.x/retrofit/retrofit2/http/QueryMap.html), which has some text like this on the webpage:

```
 &#64;GET("/search")
 Call&lt;ResponseBody> list(@QueryMap Map&lt;String, String&gt; filters);
```

I'm not sure what the proper solution is here, since I'm not a Javadoc expert. Maybe it should just be to remove the HTML entities, since [from the docs](http://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDFHHBB) it looks like `{@code}` wraps the text with `{@literal}`.

**Comment by JakeWharton** [comment]
Yeah I saw this a while ago and have been too lazy to figure out what the
deal is. I'll fix it before 2.0.2.

**Comment by dlew** [comment]
I'll take a crack at it and see what I find out.
